internal/pkg/service/common/etcdop/op/txn.go

```diff
@@ -141,10 +141,6 @@ func (v *TxnOp[R]) lowLevelTxn(ctx context.Context) (*lowLevelTxn[R], error) {
 	out := &lowLevelTxn[R]{result: v.result, client: v.client, thenOps: make([]etcd.Op, 0), elseOps: make([]etcd.Op, 0)}
 	errs := errors.NewMultiError()
 
-	// Copy processors
-	out.processors = make([]func(ctx context.Context, r *TxnResult[R]), len(v.processors))
-	copy(out.processors, v.processors)
-
 	// Copy IFs
```

Comment:
Here's another minor bug I found.

Callbacks from a nested transaction must be performed before callbacks from a parent transaction (if transactions are merged).

Because, a callback from a nested transaction can modify the result, for a callback from a parent transaction,

The error did not appear in tests because there is another bug, see next PR.

-----

So this code is moved to the end of the method, after `andOps` callbacks.